src/components/FooterWrapper.js

```diff
@@ -88,7 +95,7 @@ const FooterWrapper = ({ children, alwaysShow, fixedHeight }) => {
   const expandedFooterVisible = isExpanded || alwaysShow
 
   return div({ style: { display: 'flex', flexDirection: 'column', height: fixedHeight ? `calc(100vh - ${scrollBarHeight}px)` : '100%', flexGrow: 1 } }, [
-    div({ style: { display: 'flex', flexDirection: 'column', flexGrow: 1, height: `calc(100% - ${footerExists ? (!alwaysShow ? shrunkFooterHeight : 0) + (expandedFooterVisible ? expandedFooterHeight : 0) : 0}px)` } }, [children]),
+    div({ style: { display: 'flex', flexDirection: 'column', flexGrow: 1, height: 'max-content' } }, [children]),
```

Comment:
I'm a bit confused by this. The old calculation takes many more parameters into account and I'm not sure if `max-content` really captures the complexity of the calculation. That said, `max-content` does keep the footer anchored to the bottom of the page and I couldn't find any other pages in Terra (logged in or not) where the footer looked wrong with my change.

Does someone who understands why we were doing this calculation in the first place see a problem with using `max-content` here? It also seems like we don't really use `max-content` often in the codebase, is there a reason for that?

Reply:
@s-rubenstein looking at the git history, I think you added this code a year ago. Do you happen to recall what the height complexity was about?

Reply:
The complexity of the calculation looks related to the BioData Catalyst footer. Generally, setting the height to `100vh - footerHeight` looks like maybe at one point the footer was intended to always be visible vs scrolling out of view?

Reply:
Yes, I think we have come to the conclusion that there was an intention at one point for the footer to always be visible, even when there is a vertical scrollbar. Adam Mullen (PO) agrees with dropping that behavior and having the footer remain at the bottom of the scrollbar view in both views where the footer is present.

Reply:
I did some testing with the BioDataCatalyst (`configOverridesStore.set({ isAnalysisTabVisible: true, isBioDataCatalyst: true })`), and the change doesn't seem to impact the behavior there. FWIW, it's odd that we don't persist the "expanded" state of the footer across page views.